index.html

```diff
@@ -155,7 +189,7 @@ <h3 class="recentWorkTitle wc">
               <li class="tech twc">Ruby</li>
             </ul>
           </div>
-          <button class="mybtn spwc" type="button">See Project</button>
+          <button class="mybtn spwc projectbtn" type="button">See Project</button>
```

Comment:
- [ ] [OPTIONAL] It would be nice if you add the `cursor: pointer;` CSS attribute when you hover the "See Project" button.

![image](https://user-images.githubusercontent.com/60164239/132773936-aa2a6b91-fe4f-46f0-bde3-246cb8804f36.png)